celesta-sql/src/test/java/ru/curs/celesta/score/ParserTest.java

```diff
@@ -236,25 +239,20 @@ public void test1() throws Exception {
     g.removeIndex(idx);
     assertEquals(0, idx.getTable().getIndices().size());
 
-    t = g.getElement("employees", Table.class);
+    Table t = g.getElement("employees", Table.class);
     assertNull(t.getCelestaDoc());
     assertTrue(t.isVersioned());
-    assertFalse(t.isReadOnly());
 
-    // Проверка дополнительных возможностей
-    t = g.getElement("ttt1", Table.class);
-    assertTrue(t.isReadOnly());
-    assertFalse(t.isVersioned());
-    assertTrue(t.isAutoUpdate());
+    // Checking for additional options
+    ReadOnlyTable rot = g.getElement("ttt1", ReadOnlyTable.class);
+    assertTrue(rot.isAutoUpdate());
```

Comment:
Maybe somewhere here is the right place to check (in a separate test method) that the same object is available by calling `g.getElement("ttt1", BasicTable.class);` and `g.getElement("ttt1", ReadOnlyTable.class);`

And that   `g.getElement("employees", BasicTable.class);`/`g.getElement("employees", Table.class);` call also returns the same object.

Reply:
added.